HedgeModManager/Resources/Codesv2/SonicOrigins.hmm

```diff
@@ -541,6 +541,193 @@ Library "RSDK" by "Hyper & ĐeäTh"
     }
 }
 
+Library "UIRflParam" by "Hyper"
```

Comment:
Template libraries don't require attribution, please remove the author field.

I'd also recommend putting template libraries at the bottom of the code file for now to prioritise API libraries and codes.